### PR TITLE
Upgrade plugin to Sylius 1.13 compatibility

### PR DESCRIPTION
**Pull request by JanPalen**

### CODE REVIEW

File: src/Api/ShippingLabelFetcher.php [hunk 1]
```diff
@@ -38,13 +42,13 @@ public function createShipment($shippingGateway, $shipment): void
 
             $this->response = $this->soapClient->createShipment($requestData, $shippingGateway->getConfigValue('wsdl'));
         } catch (\SoapFault $exception) {
-            $this->flashBag->add(
+            $this->requestStack->getSession()->getBag('flashes')->add(
```
Comment (senghe) [inline src/Api/ShippingLabelFetcher.php:45]: you could assign a new variable for flashbag

File: tests/Application/config/bootstrap.php [hunk 2]
```diff
@@ -15,7 +15,7 @@
     throw new RuntimeException('Please run "composer require symfony/dotenv" to load the ".env" files configuring the application.');
 } else {
     // load all the .env files
-    (new Dotenv(true))->loadEnv(dirname(__DIR__) . '/.env');
+    (new Dotenv())->loadEnv(dirname(__DIR__) . '/.env');
```
Comment (senghe) [inline tests/Application/config/bootstrap.php]: Why this change?

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -14,30 +14,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [8.0]
-        node: [14.x]
-        symfony: [^4.4, ^5.2]
-        sylius: [~1.11.0]
-        mysql: [5.7, 8.0]
+        php: ["8.1", "8.2", "8.3"]
+        node: ["20.x"]
```
Comment (senghe) [inline .github/workflows/build.yml]: How about node 18?

**Review by senghe — CHANGES_REQUESTED** [review]

File: tests/Application/config/bootstrap.php (hunk 2 above)
Comment (JanPalen) [reply tests/Application/config/bootstrap.php]: Dotenv has changed and don't expect bool parameter. I made more research about that change. For the same effect I should use `(new Dotenv())->usePutenv()->loadEnv(dirname(__DIR__) . '/.env');`

File: .github/workflows/build.yml (hunk 3 above)
Comment (JanPalen) [reply .github/workflows/build.yml]: Sorry i pushed that before our conversation about that

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -14,30 +14,25 @@ jobs:
...
-            php: 8.0
-            mysql: 5.7
-          -
-            sylius: ~1.9.0
-            php: 8.0
-          -
-            sylius: ~1.11.0
-            php: 7.4
+          - sylius: "~1.13.0"
+            php: "8.0"
```
Comment (senghe) [inline .github/workflows/build.yml:25]: But you have no 8.0 in your matrix

File: src/Api/ShippingLabelFetcher.php [hunk 5]
```diff
@@ -57,7 +62,7 @@ public function getLabelContent(): ?string
             return '';
         }
 
-        $this->flashBag->add('success', 'bitbag.ui.shipment_data_has_been_exported'); // Add success notification
+        $this->requestStack->getSession()->getBag('flashes')->add('success', 'bitbag.ui.shipment_data_has_been_exported'); // Add success notification
```
Comment (senghe) [inline src/Api/ShippingLabelFetcher.php:64]: I mean to use one variable in all places for better readibility

**Review by senghe — CHANGES_REQUESTED** [review]

File: src/Api/ShippingLabelFetcher.php (hunk 5 above)
Comment (JanPalen) [reply src/Api/ShippingLabelFetcher.php:64]: I think there shouldn't be variable at all. There is two separate functions using that `$this->requestStack->getSession()->getBag('flashes')->add()`.

Comment (senghe) [reply src/Api/ShippingLabelFetcher.php:64]: It's not the blocker for me, but please read something about Demeter Law :ok_hand:

**Review by senghe — APPROVED** [review]
